File: 2021/Day 01/aoc01_1.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main() {
	ifstream infile;
	infile.open("aoc01_input.txt");

	int c = 0;
	int last, next;
	infile >> last;
	
	while(infile >> next) {
		if(last < next)
			c++;
		last = next;
	}
	cout << c;

	return 0;
}

File: 2025/Day 01/aoc01_1.py
file = open("aoc01_input.txt")
l = file.read().split("\n")
file.close()

s = 50
r = 0

for i in l:
    d, x = i[0], int(i[1:])

    if d == "L":
        x *= -1
    s += x

    s %= 100
    r += s == 0

print(r)